src/bindings/js/node/tests/unit/basic.test.js

```diff
@@ -27,12 +37,67 @@ describe('ov basic tests.', () => {
     modelLike = [model, compiledModel];
   });
 
+  after(() => {
+    fs.rmSync(outDir, { recursive: true });
```

Comment:
We have to wait to get gc time to release model variable before delete underlying file

```suggestion
  after(await () => {
    // Wait to ensure the model file is released
    await sleep(1);
    await fs.promises.rm(outDir, { recursive: true });
```